### PR TITLE
Cleanup, formatting and fix for multiplatform

### PR DESCRIPTION
**Pull request by Xpl0itU**
This PR adds support for other OS's by using tempfile instead of relying on an env variable for getting the temporary directory, also formats the code using the black formatter and removes an unused import

### CODE REVIEW

File: rockboxalbumfix.py [hunk 1]
```diff
@@ -197,16 +235,20 @@ def process_images(root_dir, iteration_timeout=5, max_retries=3):
     except KeyboardInterrupt:
         print("\nProcessing interrupted by user.")
 
+
 def clear_temp_directory():
-    temp_folder = os.getenv('TEMP')
-    temp_folder_path = os.path.join(temp_folder, 'cover_extraction_temp', 'cover_extraction_temp')
+    temp_folder = os.getenv("TEMP")
```
Comment (SupItsZaire) [inline rockboxalbumfix.py]: missed this small part, still uses os.getenv but will fix before pulling!

**Review by SupItsZaire — COMMENTED** [review]
accidentally missed os.getenv in 240

File: rockboxalbumfix.py (hunk 1 above)
Comment (Xpl0itU) [reply rockboxalbumfix.py]: Fixed!

**Comment by arimelody** [comment]
LGTM 👍
thank you very much!! the code readability is a great touch :)

**Comment by SupItsZaire** [comment]
thank you so much for contributing! <3

**Pull request merged**